src/rgw/rgw_rest_swift.cc

```diff
@@ -1087,6 +1087,20 @@ void RGWCopyObj_ObjStore_SWIFT::send_response()
   }
 }
 
+int RGWGetObj_ObjStore_SWIFT::verify_permission()
+{
+  op_ret = RGWGetObj_ObjStore::verify_permission();
+
+  /* We have to differentiate error codes depending on whether user is
+   * anonymous (401 Unauthorized) or he doesn't have necessary permissions
```

Comment:
@amitkumar50 Yes, please do not review backport PRs. Only review if the PR is targeting master branch. Thanks!